saga/inMemorySagaLog.go

```diff
@@ -0,0 +1,71 @@
+package saga
+
+import "errors"
```

Comment:
I have gofmt set up to run on every save. Seems like a good thing to have if your editor supports it.